src/views/Nft/market/components/Activity/ActivityPrice.tsx

```diff
@@ -14,7 +14,7 @@ const ActivityPrice = ({ bnbBusdPrice, price }) => {
             <Text maxWidth="80px" bold>
               {price.toLocaleString(undefined, {
                 minimumFractionDigits: 0,
-                maximumFractionDigits: 5,
+                maximumFractionDigits: price < 1 ? 5 : 2,
```

Comment:
Row not visible fully in iphone 8 width devices when price fraction is above 1 and has fraction